docs/application/web/api/5.5/index.htm

```diff
@@ -0,0 +1,846 @@
+								<li><a href="device_api/mobile/tizen/callhistory.html">Call History</a></li>
+								<li><a href="device_api/mobile/tizen/contact.html">Contact</a></li>
+								<li><a href="device_api/mobile/tizen/datasync.html">Data Synchronization</a></li>
+							</ul>
+						</li>
+						<li><a href="device_api/mobile/index.html#System">System</a>
+							<ul>
+								<li><a href="device_api/mobile/tizen/feedback.html">Feedback</a></li>
+								<li><a href="device_api/mobile/tizen/humanactivitymonitor.html">Human Activity Monitor</a></li>
+								<li><a href="device_api/mobile/tizen/mediakey.html">Media Key</a></li>
+								<li><a href="device_api/mobile/tizen/power.html">Power</a></li>
+								<li><a href="device_api/mobile/tizen/sensor.html">Sensor</a></li>
+								<li><a href="device_api/mobile/tizen/systeminfo.html">System Information</a></li>
+								<li><a href="device_api/mobile/tizen/systemsetting.html">System Setting</a></li>
+								<li><a href="device_api/mobile/tizen/time.html">Time</a></li>
+								<li><a href="device_api/mobile/tizen/websetting.html">Web Setting</a></li>
+							</ul>
+						</li>
+						<li><a href="device_api/mobile/index.html#UIX">UIX</a>
+							<ul>
+								<li><a href="device_api/mobile/tizen/voicecontrol.html">Voice Control</a></li>
+							</ul>
+						</li>
+						<li><a href="device_api/mobile/index.html#Cordova">Cordova</a>
+							<ul>
+								<li><a href="device_api/mobile/tizen/cordova/console.html">Console</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/cordova.html">Cordova</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/device.html">Device</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/device-motion.html">Device Motion</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/dialogs.html">Dialogs</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/events.html">Events</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/file.html">File</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/filetransfer.html">File Transfer</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/globalization.html">Globalization</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/media.html">Media</a></li>
+								<li><a href="device_api/mobile/tizen/cordova/networkInformation.html">Network Information</a></li>
+							</ul>
+						</li>
+					</ul>
+				</li>
+				<li><a href="device_api/wearable/index.html">Wearable Web</a>
+					<ul>
+						<li><a href="device_api/wearable/index.html#Base">Base</a>
+							<ul>
+								<li><a href="device_api/wearable/tizen/archive.html">Archive</a></li>
+								<li><a href="device_api/wearable/tizen/filesystem.html">Filesystem</a></li>
+								<li><a href="device_api/wearable/tizen/tizen.html">Tizen</a></li>
+							</ul>
+						</li>
+						<li><a href="device_api/wearable/index.html#Application">Application Framework</a>
```

Comment:
Compared to "5.5/device_api/wearable/index.html" file, I can see the structure doesn't match with this list.
For example we don't have "Account", "Preference", and "Social", etc for "Wearable Web" here.
Don't we need to sync this page with actual list?

Same issue to Mobile Web and TV Web.